docs/howto/system_testing.md

```diff
@@ -599,6 +600,83 @@ Finally, when you are done running all system tests, bring down the Elastic Stac
 elastic-package stack down
 ```
 
+Starting with [elastic-package version `v0.101.2`](https://github.com/elastic/elastic-package/releases/tag/v0.101.2),
```

Comment:
It is 0.101.0, right? Even if we have barely used it, it has the relevant changelog.
```suggestion
Starting with [elastic-package version `v0.101.0`](https://github.com/elastic/elastic-package/releases/tag/v0.101.0),
```